tasks/configure_namespace.yml

```diff
@@ -137,40 +137,38 @@
   - pipeline-pruner
 
 
-- name: Create clean up cronjobs
+- name: Create Pod cleaning CronJob
   k8s:
     state: present
+    namespace: "{{ osbs_ocp_namespace }}"
     definition:
-      apiVersion: batch/v1
-      kind: Job
+      apiVersion: batch/v1beta1
+      kind: CronJob
       metadata:
         name: "{{ item.name }}"
-        namespace: "{{ osbs_ocp_namespace }}"
       spec:
-        backoffLimit: 6
-        completions: 1
-        parallelism: 1
-        template:
+        # hourly
+        schedule: "* */1 * * *"
```

Comment:
I pasted this to https://crontab.guru and it says "At every minute past every hour", as in "every minute"

Reply:
Nice catch Adam .. useful tool! I changed it to `0 * * * *` hope this will reflect hourly.